dist/css/bootstrap-slider.css

```diff
@@ -30,6 +30,7 @@
   display: inline-block;
   vertical-align: middle;
   position: relative;
+  -ms-touch-action: none;
```

Comment:
could you add this to the https://github.com/seiyria/bootstrap-slider/blob/master/less/rules.less file instead?

This is a compiled file